packages/story-editor/src/components/panels/design/textStyle/fontPicker.js

```diff
@@ -57,18 +64,33 @@ const FontPicker = forwardRef(function FontPicker(
     ensureMenuFontsLoaded,
     ensureCustomFontsLoaded,
     getFontsBySearch,
+    getCustomFonts,
+    getCuratedFonts,
   } = useFont(({ actions, state }) => ({
     getFontsBySearch: actions.getFontsBySearch,
     addRecentFont: actions.addRecentFont,
     ensureMenuFontsLoaded: actions.ensureMenuFontsLoaded,
     ensureCustomFontsLoaded: actions.ensureCustomFontsLoaded,
     maybeEnqueueFontStyle: actions.maybeEnqueueFontStyle,
+    getCuratedFonts: actions.getCuratedFonts,
+    getCustomFonts: actions.getCustomFonts,
     recentFonts: state.recentFonts,
     curatedFonts: state.curatedFonts,
     fonts: state.fonts,
     customFonts: state.customFonts,
   }));
 
+  useEffect(() => {
+    mounted.current = true;
+
+    getCustomFonts(mounted);
+    getCuratedFonts(mounted);
+
+    return () => {
+      mounted.current = false;
```

Comment:
This mounted ref is not needed here

Reply:
Oh yes, it's no longer needed because the state update happens in `FontProvider` which doesn't unmount. 👍